ic-os/defs.bzl

```diff
@@ -73,6 +73,7 @@ def icos_build(
             srcs = [":copy_version_txt"],
             outs = ["version-test.txt"],
             cmd = "sed -e 's/.*/&-test/' < $< > $@",
+            visibility = ["//visibility:public"],
```

Comment:
out of curiosity, what changed here?

Reply:
Previously, the `-test` suffix was assumed in code, appended to the version from `version.txt`. Now we use this file directly.